### ISSUE TITLE
Add integration testing or otherwise figure out how best to test browser code

### ISSUE THREAD

**Issue by benogle** [benogle/electron-jasmine#2]
It seems like the cleanest way to test browser code is to start the app in a totally separate process like we're doing in atom: https://github.com/atom/atom/tree/master/spec/integration

To me, it seems the hardest part of this is figuring out the right way to download and use chromedriver
- if run standalone in a lib without electron-prebuilt, it needs to pull down chromedriver for it's electron version
- If run in an app with electron-prebuilt, it needs to pull down or use the chromedriver for the app's electron 

Could build something like (or add on to) https://github.com/maxogden/electron-download to download the correct chromedriver. 

cc @jlord @kevinsawicki

**Comment by kevinsawicki** [comment]
Yeah, https://github.com/atom/grunt-download-electron/ supports downloading chromedriver, added by @maxbrunsfeld

I think a library just for chromedriver-ing sounds good to me.

**Comment by benogle** [comment]
I wanted to avoid requiring the grunt dep, so maybe an `electron-download-chromedriver` could be the right thing.

**Comment by kevinsawicki** [comment]
> I wanted to avoid requiring the grunt dep

Sounds good
